core/src/main/resources/AppConstants.properties

```diff
@@ -270,9 +270,10 @@ ibistesttool.maxMemoryUsage=100000000
 ibistesttool.maxMessageLength=1000000
 # report transformation xslt
 ibistesttool.reportTransformation=TestTool/xsl/Default.xsl
-# file storage related properties
+# file storage related properties (when freeSpaceMinimum is not specified it is calculated as maxFileSize * (maxBackupIndex + 1) * 10)
 #ibistesttool.maxFileSize=1MB
 #ibistesttool.maxBackupIndex=9
+#ibistesttool.freeSpaceMinimum=10MB
```

Comment:
Zou je in een commentaar regel hier aan kunnen geven wat deze setting doet? Wat gebeurt er als er minder ruimte is dan aangegeven?

Reply:
Done